tools/packaging/deb/istio.mk

```diff
@@ -43,6 +44,23 @@ ISTIO_DEB_NAME ?= istio-sidecar
 # since we need configuration.
 # --iteration 1 adds a "-1" suffix to the version that didn't exist before
 ${ISTIO_OUT_LINUX}/release/istio-sidecar.deb: | ${ISTIO_OUT_LINUX} deb/fpm
+${ISTIO_OUT_LINUX}/release/istio-sidecar.rpm: | ${ISTIO_OUT_LINUX} rpm/fpm
+
+# Package the sidecar rpm file.
+rpm/fpm:
```

Comment:
Are you thinking on wiping the `rpm` dir? If not, I think this target makes more sense there, no?

Reply:
Yeah that was sort of the plan. I think its obsolete if we do this approach - is that accurate?

It is very handy to have this defined here since its shared lot of stuff with the deb. Probably to clean it up we can rename `deb` to something generic and include both